src/convnets/resnet.jl

```diff
@@ -102,25 +109,35 @@ Create a ResNet model
 - `block_config`: a list of the number of residual blocks at each stage
 - `nclasses`: the number of output classes
 """
-function resnet(block, shortcut_config::Symbol, args...; kwargs...)
-  shortcut = if shortcut_config == :A
-      (skip_identity, skip_identity)
-    elseif shortcut_config == :B
-      (skip_projection, skip_identity)
-    elseif shortcut_config == :C
-      (skip_projection, skip_projection)
-    else
-      error("Unrecognized shortcut_config ($shortcut_config) passed to `resnet` (use :A, :B, or :C).")
+function resnet(block, shortcut_config::AbstractVector{<:Symbol}, args...; kwargs...)
+  shortcut_dict = Dict(
+    :A => (skip_identity, skip_identity),
+    :B => (skip_projection, skip_identity),
+    :C => (skip_projection, skip_projection))
+
+  if any(sc -> !haskey(shortcut_dict,sc),shortcut_config)
+    error("Unrecognized shortcut_config ($shortcut_config) passed to `resnet` (use only :A, :B, or :C).")
   end
+
+  shortcut = [shortcut_dict[sc] for sc in shortcut_config]
   resnet(block, shortcut, args...; kwargs...)
 end
 
+function resnet(block, shortcut_config::Symbol, args...; block_config, kwargs...)
+    resnet(block, fill(shortcut_config, length(block_config)), args...;
+           block_config = block_config, kwargs...)
+end
+
+function resnet(block, shortcut_config::NTuple{N,Symbol}, args...; kwargs...) where N
+    resnet(block, collect(shortcut_config), args...; kwargs...)
+end
```

Comment:
Sorry, I meant for the main builder method, not the one that looks up the shortcut config.

```suggestion
resnet(block, residuals::NTuple{2}, args...; kwargs...) = resnet(block, [residuals], args...; kwargs...)
```